webapp/loggedin/surveyViews/surveyForm/nodeDefs/components/types/nodeDefBoolean.js

```diff
@@ -1,10 +1,13 @@
 import './nodeDefBoolean.scss'
 
 import React from 'react'
+import PropTypes from 'prop-types'
 
 import * as Node from '@core/record/node'
 
-const Button = props => {
+import { useI18n } from '@webapp/commonComponents/hooks'
+
+const Button = (props) => {
```

Comment:
in this context I would declare this in the render of the NodeDefBoolean component

Reply:
ok